src/docs/sphinx/user_guide/command_line_options.rst

```diff
@@ -41,6 +41,10 @@ Below is the documentation for Serac's command line options:
      - -p
      - N/A
      - Enable ParaView output
+   * - --print-unused-entries-input-file
```

Comment:
Would `--print-unused-input-file-entries` be better?

It's also rather lengthy, though very descriptive. Maybe `--print-unused`?

Thoughts @jamiebramwell ?

Reply:
I think `--print-unused` is verbose enough.